builtin/game/item.lua

```diff
@@ -514,7 +514,7 @@ function core.node_dig(pos, node, digger)
 		.. node.name .. " at " .. core.pos_to_string(pos))
 
 	local wielded = digger and digger:get_wielded_item()
-	local drops = core.get_node_drops(node, wielded and wielded:get_name())
+	local drops = core.get_node_drops(node, wielded and wielded:get_name(), ItemStack(wielded), digger, pos)
```

Comment:
```suggestion
	local drops = core.get_node_drops(node, wielded and wielded:get_name(), ItemStack(wielded), digger, vector.copy(pos))
```

same reason why you're doing `ItemStack(wielded)`